packages/@lwc/engine-core/src/framework/modules/static-parts.ts

```diff
@@ -102,13 +103,23 @@ export function mountStaticParts(root: Element, vnode: VStatic, renderer: Render
 
     // Currently only event listeners and refs are supported for static vnodes
     for (const part of parts) {
-        // Event listeners only need to be applied once when mounting
-        applyEventListeners(part, renderer);
-        // Refs must be updated after every render due to refVNodes getting reset before every render
-        applyRefs(part, owner);
-        patchAttributes(null, part, renderer);
-        patchClassAttribute(null, part, renderer);
-        patchStyleAttribute(null, part, renderer, owner);
+        if (isVStaticPartElement(part)) {
+            // Event listeners only need to be applied once when mounting
+            applyEventListeners(part, renderer);
+            // Refs must be updated after every render due to refVNodes getting reset before every render
+            applyRefs(part, owner);
+            patchAttributes(null, part, renderer);
+            patchClassAttribute(null, part, renderer);
+            patchStyleAttribute(null, part, renderer, owner);
+        } else if (isVStaticPartText(part)) {
+            patchTextVStaticPart(null, part, renderer);
+        } else {
+            if (process.env.NODE_ENV !== 'production') {
+                throw new Error(
+                    `LWC internal error, encountered unknown static part type: ${part.type}`
+                );
+            }
+        }
```

Comment:
```suggestion
        } else {
            if (process.env.NODE_ENV !== 'production' && !isVStaticPartText(part)) {
                throw new Error(
                    `LWC internal error, encountered unknown static part type: ${part.type}`
                );
            }
            patchTextVStaticPart(null, part, renderer);
        }
```

Avoid the extra `if` test in prod.